src/shared/components/complex/gokb-table/gokb-table.vue

```diff
@@ -154,6 +154,18 @@
           >
             mdi-check-bold
           </v-icon>
+          <a
+            v-if="!!item.extlink"
+            :href="item.extlink"
+          >
+            <v-icon
+              style="cursor:pointer"
+              :title="$t('btn.linkext')"
+              small
+            >
+              mdi-open-in-new
+            </v-icon>
+          </a>
```

Comment:
I think it would be best to put the external link at the start of all actions instead of last, so the first one you reach once you look right from the value itself.